src/object-list.scss

```diff
@@ -0,0 +1,210 @@
+
+@import "./new-settings";
+@import "./mixins";
+@import "./list";
+
+$block: #{$fd-namespace}-object-list;
+
+.#{$block} {
+
+  $fd-object-list-border-bottom: 1px solid var(--sapList_BorderColor);
+  $fd-object-list-with-value-zero: 0;
+  $fd-object-list-with-padding-value-half: 0.5rem;
+  $fd-object-list-with-padding-value-one: 1rem;
+  $fd-object-list-with-value-half: 0.75rem;
+  $fd-object-list-align-left: left;
+  $fd-object-list-align-right: right;
+  $fd-object-list-with-vertical-align-top: top;
+  $fd-object-list-with-vertical-align-bottom: bottom;
+  $fd-object-list-font-color: var(--sapContent_LabelColor);
+
+  &__item {
+    @include padding-top-bottom(1rem);
+
+    height: 100%;
+    width: 100%;
+    display: block;
+    box-sizing: border-box;
+    border-bottom: $fd-object-list-border-bottom;
+
+    >[class*=__link] {
+      @include fd-active() {
+        span,
+        p,
+        img,
+        i[class*=sap-icon] {
+          color: var(--sapList_Active_TextColor);
+          text-shadow: none;
+        }
+      }
+    }
+  }
+
+  &__intro {
+    @include fd-reset();
+
+    width: 100%;
+    display: flex;
+    padding-bottom: $fd-object-list-with-value-half;
+
+    &-text {
+      @include fd-ellipsis();
+
+      max-width: calc(100% - 1rem);
+      color: $fd-object-list-font-color;
+    }
+  }
+
+  &__container {
+    @include fd-reset();
+    @include fd-flex(column);
+
+    height: 100%;
+    width: 100%;
+  }
+
+  &__header {
+    display: flex;
+    vertical-align: $fd-object-list-with-vertical-align-top;
```

Comment:
This is used only in 1 place, no need to be a variable. We use variables if something is used in multiple places so in case something change in future it will be easier to update only one value